indico/modules/events/timetable/templates/pdf/_cover_page.html

```diff
@@ -0,0 +1,27 @@
+{% from 'events/display/conference/_util.html' import format_event_date %}
+
+{% macro render_cover_page(event) %}
+    <div class="pagebreak cover">
+        <div class="cover-wrapper">
+            <div>
+                {% if event.logo %}
+                    <img alt="Event logo" src="event://logo" class="logo" />
```

Comment:
Unless weasyprint requires XHTML-style self-closing tags, let's avoid them since we don't use them elsewhere.


```suggestion
                    <img alt="Event logo" src="event://logo" class="logo">
```